include/mbedtls/check_config.h

```diff
@@ -551,6 +551,16 @@
 #error "MBEDTLS_PK_WRITE_C defined, but not all prerequisites"
 #endif
 
+#if defined(MBEDTLS_PK_PARSE_C) && defined(MBEDTLS_PK_HAVE_ECC_KEYS) && \
+    !defined(MBEDTLS_OID_C)
+#error "MBEDTLS_PK_PARSE_C and MBEDTLS_PK_HAVE_ECC_KEYS require MBEDTLS_OID_C"
+#endif
+
+#if defined(MBEDTLS_PK_WRITE_C) && defined(MBEDTLS_PK_HAVE_ECC_KEYS) && \
+    !defined(MBEDTLS_OID_C)
+#error "MBEDTLS_PK_WRITE_C and MBEDTLS_PK_HAVE_ECC_KEYS require MBEDTLS_OID_C"
+#endif
```

Comment:
General rule: every time we add a check in `check_config.h`, we want to add corresponding documentation to `mbedtls_config.h`.